docs/src/main/asciidoc/user-guide/topics/exporting-mappings-to-an-atlasmap-catalog-file.adoc

```diff
@@ -4,21 +4,15 @@
 After all mappings are defined, save the mappings by exporting
 them to an AtlasMap `.adm` catalog file:
 
-. In the AtlasMap top tool bar, click 
-image:ExportIcon.png[Export]. 
+. Click the *AtlasMap* menu and *Export the current mappings and support files into a catalog (.adm)*
 +
-This captures your workspace in a `.adm` catalog file. 
+image:AtlasMapMenu.png[AtlasMap menu].
++
```

Comment:
You can't select a folder to put the .adm file into.  Web-based apps shouldn't wrote into user-defined folders.  It always writes the adm file into the Downloads folder.

Reply:
I see now that is correct, but I'm confused. Why do we allow the user to select a folder in VSC and not in standalone?